securedrop/tests/test_i18n.py

```diff
@@ -133,6 +166,9 @@ def test_i18n(self):
         pybabel init -i {d}/messages.pot -d {d} -l zh_Hans_CN
         sed -i -e '/code hello i18n/,+1s/msgstr ""/msgstr "code chinese"/' \
               {d}/zh_Hans_CN/LC_MESSAGES/messages.po
+        pybabel init -i {d}/messages.pot -d {d} -l nb_NO
```

Comment:
Pedantic: Could you add a newline here since this is a slight wall of text?

Reply:
Done. Indeed less confusing.